airgap/uds-dev-stack/zarf.yaml

```diff
@@ -0,0 +1,141 @@
+          - cmd: ./zarf tools registry pull --platform linux/amd64 cgr.dev/chainguard/wolfi-base:latest wolfi-base-amd64.tar
+            description: Pull the 'wolfi-base' amd64 image
+          - cmd: ./zarf tools registry pull --platform linux/arm64 cgr.dev/chainguard/wolfi-base:latest wolfi-base-arm64.tar
+            description: Pull the 'wolfi-base' arm64 image
+          - cmd: ./zarf tools registry pull --platform linux/amd64 registry.k8s.io/pause:3.10 pause-amd64.tar
+            description: Pull the 'pause' amd64 image
+          - cmd: ./zarf tools registry pull --platform linux/arm64 registry.k8s.io/pause:3.10 pause-arm64.tar
+            description: Pull the 'pause' arm64 image
+          - cmd: ./zarf tools registry pull --platform linux/amd64 quay.io/nginx/nginx-unprivileged:1.28.0-alpine3.21 nginx-amd64.tar
+            description: Pull the 'nginx' amd64 image
+          - cmd: ./zarf tools registry pull --platform linux/arm64 quay.io/nginx/nginx-unprivileged:1.28.0-alpine3.21 nginx-arm64.tar
+            description: Pull the 'nginx' arm64 image
+          - cmd: ./zarf tools registry pull --platform linux/amd64 quay.io/metallb/controller:v0.14.9 metallb-controller-amd64.tar
+            description: Pull the 'metallb-controller' amd64 image
+          - cmd: ./zarf tools registry pull --platform linux/arm64 quay.io/metallb/controller:v0.14.9 metallb-controller-arm64.tar
+            description: Pull the 'metallb-controller' arm64 image
+          - cmd: ./zarf tools registry pull --platform linux/amd64 quay.io/metallb/speaker:v0.14.9 metallb-speaker-amd64.tar
+            description: Pull the 'metallb-speaker' amd64 image
+          - cmd: ./zarf tools registry pull --platform linux/arm64 quay.io/metallb/speaker:v0.14.9 metallb-speaker-arm64.tar
+            description: Pull the 'metallb-speaker' arm64 image
+          - cmd: ./zarf tools registry pull --platform linux/amd64 quay.io/frrouting/frr:9.1.0 frr-amd64.tar
+            description: Pull the 'frr' amd64 image
+          - cmd: ./zarf tools registry pull --platform linux/arm64 quay.io/frrouting/frr:9.1.0 frr-arm64.tar
+            description: Pull the 'frr' arm64 image
+          - cmd: ./zarf tools registry pull --platform linux/amd64 quay.io/minio/minio:RELEASE.2025-04-03T14-56-28Z minio-amd64.tar
+            description: Pull the 'minio' amd64 image
+          - cmd: ./zarf tools registry pull --platform linux/arm64 quay.io/minio/minio:RELEASE.2025-04-03T14-56-28Z minio-arm64.tar
+            description: Pull the 'minio' arm64 image
+          - cmd: ./zarf tools registry pull --platform linux/amd64 quay.io/minio/mc:RELEASE.2024-11-21T17-21-54Z minio-mc-amd64.tar
+            description: Pull the 'minio-mc' amd64 image
+          - cmd: ./zarf tools registry pull --platform linux/arm64 quay.io/minio/mc:RELEASE.2024-11-21T17-21-54Z minio-mc-arm64.tar
+            description: Pull the 'minio-mc' arm64 image
```

Comment:
we can use `###ZARF_PKG_ARCH###` on these too (its a create template so will be in here before the package is made

Reply:
does that solve the arch issue with flavors?

Reply:
The issue is actually because of the "files" block and not being able to do a "yolo" with only cluster arch. Basically I have to include all images for all arches because we can't do component decisions based on arch in this package.